recipes/qxlsx/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "cci.20221127":
+    url: "https://github.com/QtExcel/QXlsx/archive/99c2b9d226a7da6b06cde181d28b888b008db667.zip"
+    sha256: "9d80deb17d2c0837445ec9d04558b5535f229e886d6d6607b44b8cf6e9d30335"
   "1.4.4":
     url: "https://github.com/QtExcel/QXlsx/archive/refs/tags/v1.4.4.zip"
     sha256: "3efbd6f63a1ffd521c535dce7b5a5a7e9ebd23db51e6ae8e3e2eb89796e57675"
```

Comment:
latest version is from august, why a cci version? cci versions disturbe how versions are displayed in conan center ui or conan client, it's the last resort.

Reply:
Ok I got it. There is an issue with Qt 6 on Windows that is fixed here: https://github.com/QtExcel/QXlsx/pull/241

I changed this PR to rather include this patch.